arch/arm/core/aarch64/vector_table.S

```diff
@@ -57,12 +57,10 @@ _ASM_FILE_PROLOGUE
  */
 
 	/* The whole table must be 2K aligned */
-	.align 11
-
 SECTION_SUBSEC_FUNC(exc_vector_table,_vector_table_section,_vector_table)
 
 	/* Current EL with SP0 / Synchronous */
-	.align 7
+	.align 11
```

Comment:
This is ok but I prefer leaving both `align 11` and `align 7` for the actual vector.

Reply:
@carlocaione  you mean the following?
`.align 11`
`.align 7`

Reply:
yes please, so that we can eventually move all the vector allocations in a macro some day.